### ISSUE TITLE
Make compacted item listener check lore

### ISSUE THREAD

**Issue by Maxopoly** [TealNerd/CivGuide#17]
There are other lored items than just compacted ones, especially with contraptions and AqPvP, the listener here should compare the concrete lore to ensure it's a compacted item.

**Comment by TealNerd** [comment]
yeah it can do that but it also checks in the sendEventMessage method because it has a trigger item

**Comment by Maxopoly** [comment]
I didnt look at the code, I just noticed that it triggered for every lored item

**Comment by TealNerd** [comment]
well it's irrelevant now since rourke had me implement into CivMenu
